src/map/grades.cpp

```diff
@@ -136,6 +136,34 @@ std::array<std::array<float, 4>, 8> StatScale = { {
  *                                                                        *
  ************************************************************************/
 
+std::array<std::array<float, 3>, 8> MobHPScale = { {
+    // Base | Job/SJ Scale | ScaleX
+    { 0, 0, 0 },  // 0
+    { 36, 9, 1 }, // A
+    { 33, 8, 1 }, // B
+    { 32, 7, 1 }, // C
+    { 29, 6, 0 }, // D
+    { 27, 5, 0 }, // E
+    { 24, 4, 0 }, // F
+    { 22, 3, 0 }, // G
+} };
+
+/************************************************************************
+ *                                                                        *
+ *                                                                        *
```

Comment:
We could use a little description of what RBI is/means

Or remove the giant comment box